<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>967</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Öğretmen Devamsızlık ve Nöbetçi Dağıtım Sistemi</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget_oto">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_oto">
       <attribute name="title">
        <string>Dağılım Yap</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="main_layout">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="top_layout">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QComboBox" name="gun_combobox">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_tarih">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>TARİH</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="ogretmen_combobox">
              <property name="minimumSize">
               <size>
                <width>270</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="devamsizlik_ekle_button">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Devamsız Ekle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slide_parametre">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>4</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>true</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::NoTicks</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="slide_label">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Devamsız Öğretmenler:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="devamsiz_schedule">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>220</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>300</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Günün Nöbetçi Öğretmenleri</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="nobetci_listbox">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>220</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>280</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QPushButton" name="dagit_button">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Dersleri Nöbetçilere Dağıt</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="sayilari_guncelle_buton">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Nöbet Sayılarını Güncelle</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="ayar_Slider">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="ayar_label">
                  <property name="text">
                   <string>Denge</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="minimumSize">
                 <size>
                  <width>570</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>470</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="text">
                 <string>Nöbetçi Öğretmen Ders Dağılımı</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="nobetci_schedule_manuel">
                <property name="minimumSize">
                 <size>
                  <width>540</width>
                  <height>340</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>380</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="label_nobetsayisi">
                <property name="minimumSize">
                 <size>
                  <width>174</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>320</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="text">
                 <string>Nöbet Dağılım Sayısı:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="table_nobetsayisi">
                <property name="minimumSize">
                 <size>
                  <width>170</width>
                  <height>340</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="minimumSize">
                 <size>
                  <width>182</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="text">
                 <string>Dağıtılamayan Dersler:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="dagitilamayan_table">
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>340</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="temizle_button">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>28</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Listeyi Temizle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="excel_raporu_yaz_btn">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>28</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Excel Raporla</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="data_yaz_btn">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>28</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Dağılımı Kaydet</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_manuel">
       <attribute name="title">
        <string>Dağılım Raporu</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QPushButton" name="denekButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Excel Raporu Oluştur Göster</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QTableWidget" name="tbl_sonuc">
          <property name="minimumSize">
           <size>
            <width>302</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>442</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Dağıtılan Dersler:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
